docs/ssoUpgrade/sso-7.4-upgrade-notification.md

```diff
@@ -0,0 +1,39 @@
+
+### What is happening?
+
+We are upgrading our RedHat SSO Services from 7.3 to 7.4, which is KeyCloak 4.8 to 9.0.
+- This is a major version upgrade, please see the release note for details: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html-single/release_notes/index
+- Here are the new SSO documentations: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/
+- This upgrade will also include some BCGov login theme bug fixes: https://github.com/bcgov/ocp-sso/issues/77
+
+
+### When?
+
```

Comment:
```suggestion

The planning is for an accelerated deployment cadence in order to remove blockers for the OpenShift 4 migration.  This will require fast action from our development teams, both from the initial functional validation activities, as well as notifying the platform-experience teams immediately of any critical issues identified during testing.

The full upgrade timeline is to have the upgrade rolled out to production within 3 weeks.
```